src/network/bridge.rs

```diff
@@ -652,7 +653,23 @@ fn create_interfaces(
                     data.bridge_interface_name.to_string(),
                     InfoKind::Bridge,
                 );
+
                 create_link_opts.mtu = data.mtu;
+                if create_link_opts.mtu == 0 {
+                    // if we have a default route, use its mtu as default
+                    if let Ok(iface_name) = get_default_route_interface(host) {
+                        match core_utils::get_mtu_from_iface(host, &iface_name) {
+                            Ok(iface_mtu) => {
+                                create_link_opts.mtu = iface_mtu;
+                            },
+                            Err(e) => debug!(
+                                "failed to get mtu for default interface {}: {}, using kernel default",
+                                iface_name, e
+                            ),
+                        }
+                    }
+                }
+                let mtu = create_link_opts.mtu;
```

Comment:
I think it would look a bit cleaner to define `let mut mtu = data.mtu` before the branch and then assign to mtu so we only need on `create_link_opts.mtu = mtu` assignment at the end which I find clearer. 

create_link_opts.mtu = data.mtu;